graphql_compiler/schema_generation/sqlalchemy/schema_graph_builder.py

```diff
@@ -0,0 +1,81 @@
+from collections import OrderedDict
+import warnings
+
+from graphql.type import GraphQLBoolean, GraphQLFloat, GraphQLInt, GraphQLString
+import six
+import sqlalchemy.sql.sqltypes as sqltypes
+
+from ...schema import GraphQLDate, GraphQLDateTime, GraphQLDecimal
+from ..schema_graph import PropertyDescriptor, SchemaGraph, VertexType
+
+
+# TODO(pmantica1): Add scalar mapping for the following classes: Interval, and Time.
+# We do not currently plan to add a mapping for JSON and Binary objects.
+SQL_CLASS_TO_GRAPHQL_TYPE = OrderedDict([
+    (sqltypes.String, GraphQLString),
+    (sqltypes.Integer, GraphQLInt),
+    (sqltypes.Float, GraphQLFloat),
+    (sqltypes.Numeric, GraphQLDecimal),
+    (sqltypes.DateTime, GraphQLDateTime),
+    (sqltypes.Date, GraphQLDate),
+    (sqltypes.Boolean, GraphQLBoolean),
+])
+
+
+# TODO(pmantica1): Map foreign keys to edges.
+# TODO(pmantica1): Represent table inheritance in SchemaGraph.
+# TODO(pmantica1): Add option to map tables to EdgeTypes instead of VertexTypes.
+# TODO(pmantica1): Map arrays to GraphQLLists.
```

Comment:
Do the IR, and the SQL backend support lists?

Reply:
@obi1kenobi ?

Reply:
The IR has no problem with list-typed values. I think not all SQL databases support array-typed columns, though, so there's a bit more work to be done before we can compile those successfully in the backend.

Reply:
Updated TODO statement to account for SQL array compiler targeting not being ready.